be/src/service/starrocks_main.cpp

```diff
@@ -270,6 +274,7 @@ int main(int argc, char** argv) {
     }
 
 #ifdef USE_STAROS
+    staros::starlet::fslib::register_builtin_filesystems();
```

Comment:
this can be put into StarletFileSystem constructor, it can be called multiple times with no side effect.